eventuate-tram-optimistic-locking/build.gradle

```diff
@@ -0,0 +1,5 @@
+dependencies {
+    compile project (":eventuate-tram-consumer-common")
+    compile "org.springframework.boot:spring-boot-starter-data-jpa:$springBootVersion"
```

Comment:
This feature is nothing to do with JPA specifically, especially since you should retry on OptimisticLockingFailureException.

I am not sure which Spring starter/artifact to depend on but it is not the JPA one

Reply:
I will check other artifacts